docs/getting-started/quickstart.md

```diff
@@ -44,6 +44,7 @@ How can we make this script schedulable, observable, resilient, and capable of r
 pip install -U prefect
 ```
 
+Note that [httpx](https://www.python-httpx.org/) is an HTTP client library and a dependency of Prefect.
```

Comment:
this feels out of place to me

![image](https://github.com/PrefectHQ/prefect/assets/31014960/27ff2c1d-599e-4925-ad78-a20166e3a1c3)

if we're in the middle of a high level description of what you need to get started with prefect, i don't think that taking a detour to explain one of our dependencies is necessary

maybe we could have a comment in the code on the line where we import it?

Reply:
That's a good thought. Will update.